pkg/landscaper/controllers/healthcheck/controller.go

```diff
@@ -114,19 +116,22 @@ func (c *lsHealthCheckController) check(ctx context.Context, log logging.Logger)
 	return lsv1alpha1.LsHealthCheckStatusOk, "ok"
 }
 
-func (c *lsHealthCheckController) checkDeployment(ctx context.Context, namespace string, name string, log logging.Logger) (bool, string) {
+func (c *lsHealthCheckController) checkDeployment(ctx context.Context, namespace string, name string) (bool, string) {
+	logger, ctx := logging.FromContextOrNew(ctx, []interface{}{lc.KeyMethod, "checkDeployment"})
+
 	key := client.ObjectKey{Namespace: namespace, Name: name}
 	deployment := &v1.Deployment{}
 	if err := c.client.Get(ctx, key, deployment); err != nil {
 		message := fmt.Sprintf("deployment %s/%s could not be be fetched", namespace, name)
-		log.Error(err, message)
+		logger.Error(err, message)
 		return false, message
 	}
 
 	if deployment.Generation != deployment.Status.ObservedGeneration ||
 		deployment.Status.UpdatedReplicas != 1 ||
 		deployment.Status.AvailableReplicas != 1 {
 		message := fmt.Sprintf("not all pods are running or at the latest state for deployment %s/%s ", namespace, name)
```

Comment:
Shouldn't namespace and name be logged by key value pairs?

Reply:
good point - I have changed it